lib/ui/index.js

```diff
@@ -4,6 +4,7 @@ var PleasantProgress = require('pleasant-progress');
 var Promise          = require('../ext/promise');
 var EOL              = require('os').EOL;
 var writeError       = require('./write-error');
+var Prompt           = require('inquirer').ui.Prompt;
```

Comment:
this is explicitly required lazily so we don't incure the penalty at cli boot for parts of the CLI that do not require inquirer